frontend/src/pages/ecommerce/fallback.tsx

```diff
@@ -1,8 +1,10 @@
-import { useMutation } from "@apollo/client";
+import { useMutation, useQuery } from "@apollo/client";
```

Comment:
This is the page we're redirected to after completing the Vipps-payment, right? Could we possibly resolve the log-in issue by redirecting users to the log in page if they're not authenticated instead of redirecting them, and utilizing the additions in #409 to redirect them back to this page after login?

Reply:
That is possible, however I don't think it is necessary, as it disrupts the user experience (they have to log in again to see their order confirmation). The log-in issue is only a case when the user is not logged in to the website on their default browser on mobile. IMO its fine that they just get the order confirmation in a logged-out state, but we can add a log-in button to give the user feedback that they are not logged in.

Reply:
Great points, I agree! I don't think adding a log-in button is necessarily desirable.